### ISSUE TITLE
Additional Hand Gestures Requested

### ISSUE THREAD

**Issue by greenwoodms06** [microsoft/MixedReality-UXTools-Unreal#40]
The hand gestures currently available in UXTools (i.e., grab and select) are great. However, some additional basic built in gestures may be very valuable to the community. Examples may include:
- single and double tap (e.g., a la air tap https://docs.microsoft.com/en-us/hololens/hololens2-basic-usage)
- side swipes (e.g., to dismiss a menu)
- other common signals such as "ok", "stop/halt"

Not sure if you are taking feature requests but thought it wouldn't hurt to ask. Thanks for all your efforts.

**Comment by luis-valverde-ms** [comment]
Thanks for the feedback @greenwoodms06 . 

You can listen for tap gestures using [UE's action mappings directly](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unreal/unreal-hand-tracking?tabs=426#gestures) (in OpenXR). If you do so, bear in mind there is a bug in UXT 0.12.0 that prevents user code from receiving those events: #41 . There's a fix for that in #39 .

We have in our backlog enabling interaction with UXT controls via [Gaze and Commit](https://docs.microsoft.com/en-us/windows/mixed-reality/design/gaze-and-commit), but have no timeline for it at the moment.

We don't have side side swipes or _ok_/_stop_ gestures in our sights at the moment. Have you seen them used anywhere (e.g. in MRTK-Unity)?

**Comment by greenwoodms06** [comment]
Regarding the mapping, I did in fact run into that action mapping bug and modified 0.12 based on those commits last week. Once I did that I implemented my own double tap using the built in grip/select using timers and got the functionality I was hoping for while avoiding overriding its default capabilities. So mission accomplished here I think.

Yup, understand the backlog. I'm sure yall have a large list of to-do's. Lot's to do to catchup to the Unity side.

Regarding side swipes, my initial thoughts would they would be really great for things like dismissing/closing messages and menus. More broadly, ok/stop gestures falls in the realm of not what already exists but if available what would folks do with them if they did exist. I would think there would be a nice procedural or ML method for generating new gestures for UXT tools using things such as standard sign language symbols/gestures. Those would seem to be a great area to look at for ideas of things that would be useful for users. Anyways, just throwing out ideas.

Much obliged for your response. Feel free to close this issue if you don't need it anymore. Cheers!
